src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Bootstrap gitops automation with Gimlet CLI
+title: Bootstrap gitops automation with Gimlet CLI using one github repository 
```

Comment:
Let's not change the title.

I see why you wanted to change it, cause we were having a consistency issue within the article.

I believe if we fix the article, the title will work okay.

Reply:
Can you change line 6 to this:

```
In this tutorial you will use the Gimlet CLI to bootstrap the gitops automation in a dummy repository. You will also write application manifests to this gitops repository and see them deploy.
```

Added `in a dummy repository` so it is perhaps more clear that it is a standalone tutorial, not following Gimlet gitops repo best practices.